plugin.video.meta/resources/lib/meta/navigation/movies.py

```diff
@@ -94,11 +94,15 @@ def movies_play_by_name(name, lang = "en"):
 
     name = urllib.unquote(name)
 
-    items = tmdb.Search().movie(query=name, language=LANG, page=1)["results"]
+    items = tmdb.Search().movie(query=name, language=lang, page=1)["results"]
+
+    if items == []:
+        dialogs.ok(_("Movie not found"), "no movie information found for {0} in tmdb".format(name))
+        return
 
     if len(items) > 1:
-        selection = dialogs.select(_("Choose Movie"),
-                                   [to_utf8(s["title"]) + " (" + parse_year(s["release_date"]) + ")" for s in items])
+        selection = dialogs.select(_("Choose Movie"), ["{0} ({1})".format(
+            unicode.encode(s["title"], "ascii", "ignore"), parse_year(s["release_date"])) for s in items])
```

Comment:
using to_utf8 sometimes errors out with
14:34:40 T:556   ERROR: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--
                                             - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
                                            Error Type: <type 'exceptions.UnicodeDecodeError'>
                                            Error Contents: 'ascii' codec can't decode byte 0xc3 in position 18: ordinal not in range(128)
                                            Traceback (most recent call last):
                                              File "D:\appdata\roaming\Kodi\addons\plugin.video.meta\addon.py", line 177, in <module>
                                                main()
                                              File "D:\appdata\roaming\Kodi\addons\plugin.video.meta\addon.py", line 174, in main
                                                plugin.run()
                                              File "D:\appdata\roaming\Kodi\addons\plugin.video.meta\resources\lib\xbmcswift2\plugin.py", line 333, in run
                                                items = self._dispatch(self.request.path)
                                              File "D:\appdata\roaming\Kodi\addons\plugin.video.meta\resources\lib\xbmcswift2\plugin.py", line 306, in _dispatch
                                                listitems = view_func(**items)
                                              File "D:\appdata\roaming\Kodi\addons\plugin.video.meta\resources\lib\meta\navigation\movies.py", line 106, in movies_play_by_name
                                                [to_utf8(s["title"]) + " (" + parse_year(s["release_date"]) + ")" for s in items])
                                            UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 18: ordinal not in range(128)
                                            -->End of Python script error report<--